docs/react-testing-library/setup.mdx

```diff
@@ -28,37 +31,71 @@ make your test util file accessible without using relative paths.
 The example below sets up data providers using the [`wrapper`](api.mdx#wrapper)
 option to `render`.
 
-```diff title="my-component.test.js"
-- import { render, fireEvent } from '@testing-library/react';
-+ import { render, fireEvent } from '../test-utils';
-```
-
-```js title="test-utils.js"
-import React from 'react'
-import { render } from '@testing-library/react'
-import { ThemeProvider } from 'my-ui-lib'
-import { TranslationProvider } from 'my-i18n-lib'
-import defaultStrings from 'i18n/en-x-default'
-
-const AllTheProviders = ({ children }) => {
-  return (
-    <ThemeProvider theme="light">
-      <TranslationProvider messages={defaultStrings}>
-        {children}
-      </TranslationProvider>
-    </ThemeProvider>
-  )
-}
-
-const customRender = (ui, options) =>
-  render(ui, { wrapper: AllTheProviders, ...options })
-
-// re-export everything
-export * from '@testing-library/react'
-
-// override render method
-export { customRender as render }
-```
+<Tabs groupId="test-utils" defaultValue="js" values={[ {label: 'Javascript',
+value: 'js'}, {label: 'Typescript', value: 'tsx'}, ]}>
+
+  <TabItem value="js">
+
+    ```jsx title="test-utils.js"
```

Comment:
I guess this should be: `test-utils.jsx` since we do use jsx in this file (same as with tsx).
Also, I think that in the process you've removed the diff part:
```diff title="my-component.test.js"
- import { render, fireEvent } from '@testing-library/react';
+ import { render, fireEvent } from '../test-utils';
```